src/hub.ts

```diff
@@ -25,15 +27,20 @@ export function handleSignup(event: SignupEvent): void {
 }
 
 export function handleTrust(event: TrustEvent): void {
```

Comment:
Nice with deleting the entry when the limit is set to 0!

Will this store two trust connections in the database if I send a request from A to B with limit 12 and then later again with 20?

The wanted behaviour is maybe that it overwrites the old limit instead of creating a new entry.